src/locales/en/translation.json

```diff
@@ -92,6 +93,12 @@
     "valuePair": "{{value, number}}"
   },
   "contract": {
+    "code": "Code",
+    "copyButton": "Copy {{subject}}",
+    "creationByteCode": "Creation ByteCode",
+    "noCode": "There is no bytecode on record for this account. (Are you sure this is a contract?)",
```

Comment:
I would remove `(Are you sure this is a contract?)` part.

Reply:
A bit of explanation here: all contracts have at least runtime bytecode. So the only way to _not_ see any code is to look at something that is not a contract. However, in that situation, the "code" tab won't even be displayed. 

So in order to see this, you have to manually hack the URL and add `/code` to the end of it, which is a strongly deliberate action. I assume it won't often happen, and when it will, it will be the result of some very deliberate user action. That is why I decided that it's appropriate to give a more poignant error message in this case.

Do you still think it should be removed?

Reply:
whether we need it or not is up to you. if you see it valuable I am fine with it.